src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,69 @@
+package pages;
+
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.PageFactory;
+
+public class LoginPage {
+
+    public LoginPage(WebDriver driver) {
```

Comment:
Please use your IDE's code formatter to avoid these kind of line breakings. It is easier to read the code if the formatting is consistent.